pkg/chunk/grpc/grpc.pb.go

```diff
@@ -7,17 +7,18 @@ import (
 	bytes "bytes"
 	context "context"
 	fmt "fmt"
+	io "io"
+	math "math"
+	math_bits "math/bits"
+	reflect "reflect"
+	strings "strings"
+
```

Comment:
**nit:** Not sure you meant to commit this. This looks like a autoformatter hitting a generated file.

Reply:
Yes that was a too aggressive `go-imports` run